core/src/main/resources/defaults/clustertemplates/7.2.15/aws/dataengineering-ha.json

```diff
@@ -87,12 +87,12 @@
               "strategy" : "PARTITION"
             }
           },
-          "instanceType": "m5.4xlarge",
+          "instanceType": "r5d.4xlarge",
           "rootVolume": {
             "size": 150
```

Comment:
Why is the root volume size changed in the 7.2.14 template but not in the 7.2.15?